mmv1/products/dlp/JobTrigger.yaml

```diff
@@ -158,6 +175,413 @@ properties:
+                                    name: 'windowAfter'
+                                    description: |
+                                      Number of characters after the finding to consider. Either this or window_before must be specified
+          - !ruby/object:Api::Type::Array
+            name: 'customInfoTypes'
+            description: |
+              Custom info types to be used. See https://cloud.google.com/dlp/docs/creating-custom-infotypes to learn more.
+            item_type: !ruby/object:Api::Type::NestedObject
+              properties:
+                - !ruby/object:Api::Type::NestedObject
+                  name: 'infoType'
+                  required: true
+                  description: |
+                    CustomInfoType can either be a new infoType, or an extension of built-in infoType, when the name matches one of existing
+                    infoTypes and that infoType is specified in `info_types` field. Specifying the latter adds findings to the
+                    one detected by the system. If built-in info type is not specified in `info_types` list then the name is
+                    treated as a custom info type.
+                  properties:
+                    - !ruby/object:Api::Type::String
+                      name: 'name'
+                      required: true
+                      description: |
+                        Name of the information type. Either a name of your choosing when creating a CustomInfoType, or one of the names
+                        listed at https://cloud.google.com/dlp/docs/infotypes-reference when specifying a built-in type.
+                    - !ruby/object:Api::Type::String
+                      name: 'version'
+                      description: |
+                        Version of the information type to use. By default, the version is set to stable.
+                - !ruby/object:Api::Type::Enum
+                  name: 'likelihood'
+                  description: |
+                    Likelihood to return for this CustomInfoType. This base value can be altered by a detection rule if the finding meets the criteria
+                    specified by the rule.
+                  values:
+                    - :VERY_UNLIKELY
+                    - :UNLIKELY
+                    - :POSSIBLE
+                    - :LIKELY
+                    - :VERY_LIKELY
+                  default_value: :VERY_LIKELY
+                - !ruby/object:Api::Type::Enum
+                  name: 'exclusionType'
+                  description: |
+                    If set to EXCLUSION_TYPE_EXCLUDE this infoType will not cause a finding to be returned. It still can be used for rules matching.
+                  values:
+                    - :EXCLUSION_TYPE_EXCLUDE
+                - !ruby/object:Api::Type::NestedObject
+                  name: 'regex'
+                  description: Regular expression which defines the rule.
+                  immutable: true
```

Comment:
Is `immutable: true` is required here?

Reply:
I've checked this field and it runs fine without `ForceNew`. Hence, removing `immutable:true`.